jdaviz/configs/default/plugins/plot_options/plot_options.vue

```diff
@@ -348,6 +348,15 @@
 
     <v-row v-if="stretch_function_sync.in_subscribed_states">
       <!-- z-index to ensure on top of the jupyter widget with negative margin-top -->
+      <v-text-field
+          ref="stretch_hist_nbins"
+          type="number"
+          label="Bin Size"
+          v-model.number="stretch_hist_nbins"
+          hint="The amount of bins used in the histogram."
+          persistent-hint
+          :rules="[() => stretch_hist_nbins > 0 || 'Bin size must be greater than zero']"

```

Comment:
verification for empty string here would also be helpful.  When it is empty, should the change be ignored, the entire plot clear, or just the vertical marks?

Reply:
My opinion is the change should be ignored on empty string. Do you know off the top of your head what the code would look?

Reply:
Ok, works for me - I think just adding another rule here, change to using `IntHandleEmpty` to avoid the traceback, and then in `set_histogram_bin_size` have a `if self.histogram_bin_size == '': return`.

Reply:
Right, I have everything else ready to go. Just wondering what the rule would look like.

Reply:
`() => !!stretch_hist_nbins || 'Number of bins is required'` ?